b827ebfffe2fb98b

```diff
@@ -0,0 +1,18 @@
+{
+   "gateway_conf": {
+     "gateway_ID": "b827ebfffe2fb98b",
```

Comment:
Please uppercase the ID, both here and in the filename, and also add the `.json` extension to the file, otherwise it won't work. Thanks!